test/ruby/test_allocation.rb

```diff
@@ -277,7 +277,7 @@ def self.keyword(a: nil#{block}); end
         check_allocations(0, 1, "keyword(**hash1, **empty_hash#{block})")
```

Comment:
Just curious is it robust for the finalizer which allocates objects? 
I don't see such failure though.

Reply:
It uses `assert_separately`, so hopefully finalizers that allocate shouldn't be an issue.